lib/internal/errors.js

```diff
@@ -1080,7 +1080,8 @@ E('ERR_INVALID_ARG_VALUE', (name, value, reason = 'is invalid') => {
   if (inspected.length > 128) {
     inspected = `${inspected.slice(0, 128)}...`;
   }
-  return `The argument '${name}' ${reason}. Received ${inspected}`;
+  const type = name.includes('.') ? 'property' : 'argument';
+  return `The ${type} '${name}' ${reason}. Received ${inspected}`;
```

Comment:
While changing this, we might also want to change the order to:

```suggestion
  return `The '${name}' ${type} ${reason}. Received ${inspected}`;
```

This seems nicer to read to me.

Reply:
Looks better to me as well, let's ping a few more people since this incurs lots of test changes
@Trott @jasnell @addaleax @mcollina wdyt?